humanoid_nmpc/humanoid_common_mpc/src/common/ModelSettings.cpp

```diff
@@ -50,16 +52,19 @@ namespace {
  * @brief Creates a joint Index map from a list of joint names.
  */
 
-static std::unordered_map<std::string, size_t> createJointIndexMap(const std::vector<std::string>& jointNames, size_t offset = 0) {
+static std::unordered_map<std::string, size_t>
```

Comment:
I think we should enforce some formatting on the repo. I think we use an almoust identical formatter. I sticked with the ocs2 one. Would you mind reformatting with?

 "clang-format.style": "{ BasedOnStyle: Google, ColumnLimit: 140, IndentWidth: 2, BreakBeforeBraces: Attach, DerivePointerAlignment: false, PointerAlignment: Left, AllowShortFunctionsOnASingleLine: Inline, IncludeBlocks: Preserve, BinPackParameters: false}",